docs(QTM-698): Inclusion in the docs of the suggestion not to use MUI in implem (#559)
catho/quantum: CONTRIBUTING.MD

Why _commitizen_? To enforce a padronization to the commits and make sure every commit follows the same profile.

---
## Important Notices ⚠️ 

We do not recommend using ready-made components from [MUI](https://mui.com/material-ui/getting-started/) when implementing Quantum components for performance and long-term maintenance reasons. MUI is included in our library only to enable the use of [Material Icons](https://mui.com/material-ui/material-icons/).

> The MUI library, as well as the Styled Components, internally use resources and libraries that are an impediment to the use of [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource included in Next JS 13 that we intend to enable its use and consequently improve the performance of our components. In the future, we plan to remove the cited libraries completely.

Reviewer: The MUI library, as well as the Styled Components, uses internally resources and libraries that aren't compatible with [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource added to Next JS 13. We intend to enable its use in the future to improve the performance of our components. Therefore, we plan to remove the mentioned libraries completely from our components.

> The MUI library, as well as the Styled Components, internally use resources and libraries that are an impediment to the use of [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource included in Next JS 13 that we intend to enable its use and consequently improve the performance of our components. In the future, we plan to remove the cited libraries completely.

Reviewer: The MUI library, as well as the Styled Components, uses internally resources and libraries that aren't compatible with [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource added to Next JS 13. We intend to enable its use in the future to improve the performance of our components. Therefore, we plan to remove the mentioned libraries completely from our components.

> The MUI library, as well as the Styled Components, internally use resources and libraries that are an impediment to the use of [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource included in Next JS 13 that we intend to enable its use and consequently improve the performance of our components. In the future, we plan to remove the cited libraries completely.

Reviewer: ```suggestion
The MUI library, as well as the Styled Components, uses internally resources and libraries that aren't compatible with [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource added to Next JS 13. We intend to enable its use in the future to improve the performance of our components. Therefore, we plan to remove the mentioned libraries completely from our components.
```

> We also do not recommend using any other ready-made component libraries. We suggest that new components have their DOM structure, styling and functionalities completely implemented internally, that is, without dependence on third-party libraries.

Reviewer: Also, we do not recommend using any other ready-made component library. We suggest for new components to have DOM structure, styling and functionalities completely implemented internally, without any dependence on third-party libraries.

> We also do not recommend using any other ready-made component libraries. We suggest that new components have their DOM structure, styling and functionalities completely implemented internally, that is, without dependence on third-party libraries.

Reviewer: ```suggestion
Also, we do not recommend using any other ready-made component library. We suggest for new components to have DOM structure, styling and functionalities completely implemented internally, without any dependence on third-party libraries.
```
